### ISSUE TITLE
Add "Surefire Migration Support" section to README.md

### ISSUE THREAD

**Issue by sormuras** [sormuras/junit-platform-maven-plugin#41]

**Comment by jorsol** [comment]
It would be nice to add in the README, the advantages over surefire, right now it's not clear why should we substitute surefire with junit-platform-maven-plugin beyond modular testing.

**Comment by rmannibucau** [comment]
The main advantage is the cleaner plugin (only runs an engine, no abstraction, direct bridge to the JVM IO instead of a (unix) socket bridge, less code so easier to investigate in case of issue and a closer design to junit5 instead of limitation of the surefire abstraction layer). The drawback is you don't get all the surefire integrations OOTB (reporting etc) but you need to do it yourself based on xml reports for example.

@jorsol do you want to draft a PR around it?

**Comment by jorsol** [comment]
Yes, I would like to draft a PR around it, but I'm not familiar enough with the inner workings of either so I might not be the best for it.
